client/views/vote.html

```diff
@@ -12,7 +12,7 @@
         <div ng-if="updated">Vote Accepted</div>
       </form>
       <div class="">
-        <div class="actionBox">
+        <div class="actionBox" ng-if="imageId !== -1">
```

Comment:
Could you retract this change?  My pull request addresses this page and handles this.